client/resource_group/controller/controller.go

```diff
@@ -843,10 +861,9 @@ func (gc *groupCostController) modifyTokenCounter(counter *tokenCounter, bucket
 		cfg.NewTokens = granted
 		cfg.NewRate = float64(bucket.GetSettings().FillRate)
 		counter.lastDeadline = time.Time{}
-		cfg.NotifyThreshold = math.Min(granted+counter.limiter.AvailableTokens(gc.run.now), counter.avgRUPerSec*float64(defaultTargetPeriod)) * notifyFraction
-		// In the non-trickle case, clients can be allowed to accumulate more tokens.
-		if cfg.NewBurst >= 0 {
-			cfg.NewBurst = 0
+		cfg.NotifyThreshold = math.Min((granted+counter.limiter.AvailableTokens(gc.run.now)), counter.avgRUPerSec*defaultTargetPeriod.Seconds()) * notifyFraction
+		if cfg.NewBurst < 0 {
```

Comment:
```suggestion
		if cfg.NewBurst <= 0 {
```
if NewBurst is zero, is it reasonable to reset the tokens  in client side?

Reply:
It's not reasonable. if NewBurst is zero, the tokens of the client should be allocated by the server